<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterForm</class>
 <widget class="QWidget" name="ParameterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>参数设置</string>
   </property>
   <widget class="QLabel" name="labelGPIBSelector">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>GPIB设备选择</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboGPIBSelector">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLock">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Lock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUnlock">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Unlock</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIdleCurrent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>关机电流</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWorkCurrent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>开机电流</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelChargeCurrent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>充电电流</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinCurrent">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxCurrent">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMinIdleCurrent">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMinWorkCurrent">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMaxIdleCurrent">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMaxWorkCurrent">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMinChargeCurrent">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMaxChargeCurrent">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelIdleCurrentUnit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>mA</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWorkCurrentUnit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>mA</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelChargeCurrentUnit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>mA</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
